data/catalogs/STRchive-disease-loci.hg38.stranger.json

```
@@ -988,14 +988,14 @@
 },
 {
   "LocusId": "FRAXE_AFF2",
-  "ReferenceRegion": "chrX:148500604-148500753",
+  "ReferenceRegion": "chrX:148500637-148500682",
   "LocusStructure": "(GCC)*",
   "VariantType": "Repeat",
   "HGNCId": null,
   "InheritanceMode": ["XR"],
   "DisplayRU": "GCC",
   "Disease": "FRAXE",
-  "NormalMax": 39,
+  "NormalMax": 69,
```

And this appears to have transferred from the previous PR. Kind bot, but not as intended.
```suggestion
  "NormalMax": 39,
```